service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/NoticeOfChangeControllerSubmittedTest.java

```diff
@@ -0,0 +1,143 @@
+        .legalRepresentation("Yes")
+        .solicitor(RespondentSolicitor.builder()
+            .firstName("Other")
+            .lastName("Solicitor")
+            .email("other@test.com")
+            .organisation(Organisation.builder().organisationID("123").build()).build())
+        .build();
+
+    private static final CaseData CASE_DATA_BEFORE = CaseData.builder()
+        .id(CASE_ID)
+        .caseName(CASE_NAME)
+        .respondents1(wrapElements(
+            OTHER_RESPONDENT,
+            Respondent.builder()
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName("Old")
+                    .lastName("Solicitor")
+                    .email(OLD_EMAIL)
+                    .organisation(Organisation.builder().organisationID("123").build()).build())
+                .build()
+        )).build();
+
+    private static final CaseData CASE_DATA = CASE_DATA_BEFORE.toBuilder()
+        .respondents1(wrapElements(
+            OTHER_RESPONDENT,
+            Respondent.builder()
+                .legalRepresentation("Yes")
+                .solicitor(RespondentSolicitor.builder()
+                    .firstName("New")
+                    .lastName("Solicitor")
+                    .email(NEW_EMAIL)
+                    .organisation(Organisation.builder().organisationID("123").build()).build())
+                .build()))
+        .build();
+
+    @MockBean
+    private NotificationClient notificationClient;
+
+    NoticeOfChangeControllerSubmittedTest() {
+        super("noc-decision");
+    }
+
+    @Test
+    void shouldNotifyRespondentSolicitorsWhenNoticeOfChangeIsSubmitted() {
+
+        NoticeOfChangeRespondentSolicitorTemplate noticeOfChangeNewRespondentSolicitorTemplate =
+            getExpectedNoticeOfChangeParameters("Dear New Solicitor");
+
+        NoticeOfChangeRespondentSolicitorTemplate noticeOfChangeOldRespondentSolicitorTemplate =
+            getExpectedNoticeOfChangeParameters("Dear Old Solicitor");
+
+        final Map<String, Object> newSolicitorParameters = caseConverter.toMap(
+            noticeOfChangeNewRespondentSolicitorTemplate);
+
+        final Map<String, Object> oldSolicitorParameters = caseConverter.toMap(
+            noticeOfChangeOldRespondentSolicitorTemplate);
+
+        postSubmittedEvent(toCallBackRequest(CASE_DATA, CASE_DATA_BEFORE));
+
+        checkUntil(() -> {
+                verify(notificationClient).sendEmail(
+                    NOTICE_OF_CHANGE_NEW_REPRESENTATIVE,
+                    NEW_EMAIL,
+                    newSolicitorParameters,
+                    NOTIFICATION_REFERENCE);
+
+                verify(notificationClient).sendEmail(
+                    NOTICE_OF_CHANGE_FORMER_REPRESENTATIVE,
+                    OLD_EMAIL,
+                    oldSolicitorParameters,
+                    NOTIFICATION_REFERENCE);
+            }
+        );
```

Comment:
It might be worth checking that only 2 emails are sent here, since this is just checking that at least these 2 emails are triggered.